File: LLM-Translations/GPT-4/codenet/Python/Java/s964794058.java
import java.util.ArrayList;
import java.util.Scanner;

public class s964794058 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int memTime = sc.nextInt();
        sc.nextLine();
        String[] input = sc.nextLine().split(" ");
        ArrayList<Integer> memLate = new ArrayList<>();
        for (String s : input) {
            memLate.add(Integer.parseInt(s));
        }
        ArrayList<Integer> colorList = new ArrayList<>();
        int memColor = 0;
        int highLate = 0;
        for (int color = 0; color < memTime; color++) {
            int member = memLate.get(color);
            if (member / 400 >= 8) {
                highLate++;
            } else if (!colorList.contains(member / 400)) {
                colorList.add(member / 400);
            }
        }
        int maxColor = colorList.size() + highLate;
        int minColor = colorList.size();
        if (minColor == 0) {
            minColor = 1;
        }
        System.out.println(minColor + " " + maxColor);
    }
}
// 

File: LLM-Translations/GPT-4/codenet/Python/C/s964794058.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    int memTime, memLate[100], color;
    scanf("%d", &memTime);
    for(int i = 0; i < memTime; i++) {
        scanf("%d", &memLate[i]);
    }
    int colorList[10] = {0};
    int memColor = 0;
    int highLate = 0;
    for(int i = 0; i < memTime; i++) {
        int member = memLate[i];
        if (member / 400 >= 8) {
            highLate++;
        } else {
            color = member / 400;
            bool found = false;
            for(int j = 0; j < 10; j++) {
                if (colorList[j] == color) {
                    found = true;
                    break;
                }
            }
            if (!found) {
                colorList[memColor++] = color;
            }
        }
    }
    int maxColor = memColor + highLate;
    int minColor = memColor;
    if (minColor == 0) {
        minColor = 1;
    }
    printf("%d %d\n", minColor, maxColor);
    return 0;
}
// 
